webapp/middleware/isAdmin.js

```diff
@@ -1,5 +1,5 @@
 export default ({ store, error }) => {
   if (!store.getters['auth/isAdmin']) {
-    return error({ statusCode: 403 })
+    return error({ statusCode: 403, message: 'error-pages.not-authorized' })
```

Comment:
I'm having difficulties with `message` here. It's not the message, it's the translation key.

So, on the consuming side, I would check if there is a translation key for `error.message` (check if it's a key or if the error message is coming from NuxtJS). Just to make sure that we're not trying to translate a default Nuxt error message here.